CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 - General
   - Bump clj-kondo to `2023.12.16-20240202.114914-15`.
 
+- Editor
+  - Added setting to allow requires and imports to be added within the current comment form during code action and completion: `:completion {:add-libs-to-rcf [true|false|nil]}` #1316
```

Comment:
Please remember to update settings.md + docs/all-available-settings.md with the default value